docs/install.md

```diff
@@ -6,7 +6,7 @@ Choose your operating system:
 - [Installing on Mac OS X](#installing-on-mac-os-x)
 - [Installing on Windows](#installing-on-windows)
 
-If you want to create a development environment, see [docs/development.md](/docs/development.md).
+If you want to create a development environment, see [docs/development.md](https://github.com/quantumlib/Cirq/blob/master/docs/development.md).
```

Comment:
The problem with the previous one was that it had the first "/".  Instead we can just use the fact that development.md is in this  directory.  Please change the part after the comma to
`see [these instructions](development.md).`
and this should fix this.

Reply:
Wouldn't that link to https://cirq.readthedocs.io/development.md? Which also doesn't exist.